R/immune_deconvolution_methods.R

```diff
@@ -331,6 +331,8 @@ deconvolute_consensus_tme <- function(gene_expression_matrix,
     list.results[[t]] <- cur.results
   }
 
+  rownames <- Reduce(intersect, lapply(list.results, rownames))
+  list.results <- lapply(list.results, function(x) x[rownames, ])
```

Comment:
Why? And wouldn't you rather want an "outer" join with NAs (or 0s) in places where individual samples don't have a cell-type? 

Maybe add a comment, why this was necessary to understand this better in the future.